nuxeo-data-table/iron-data-table.html

```diff
@@ -671,7 +678,7 @@ <h2>[[i18n('command.add')]]</h2>
             // unreliable with Shadow, document.activeElement doesn't go inside
             // the shadow root.
             return target.contains(Polymer.dom(document.activeElement).node) ||
-                target.tagName === 'NUXEO-DATA-TABLE-CHECKBOX' || target.tagName === 'A';
+                target.tagName === this.checkboxTagName || target.tagName === 'A';
```

Comment:
that you can access your custom tag name here. And here it should be `this._tagName`

Reply:
```suggestion
                target.tagName === `${this.tagName}-CHECKBOX` || target.tagName === 'A';
```

Reply:
used _checkboxTagName